lispusers/WHO-LINE

```diff
@@ -591,18 +593,35 @@ Copyright (c) 1986-1989, 1994, 1998, 2001, 2021 by Xerox Corporation.
 
 (DEFINEQ
 
-(WHO-LINE-HOST-NAME
-(LAMBDA NIL (* ; "Edited 14-Jan-87 12:46 by smL") (* ;;; "") (* ;;; "Return the name of the curren workstation.  Avoid consing up a new string if possible.") (* ;;; "") (* ;; "") (* ;; "The cached value in *WHO-LINE-HOST-NAME* gets invalidated by an entry on the list of \SYSTEMCACHEVARS") (* ;; "") (DECLARE (GLOBALVARS *WHO-LINE-HOST-NAME*)) (if *WHO-LINE-HOST-NAME* then *WHO-LINE-HOST-NAME* else (SETQ *WHO-LINE-HOST-NAME* (ETHERHOSTNAME))))
-)
+(WHO-LINE-HOST-NAME
+  [LAMBDA NIL                                                (* ; "Edited 12-Apr-2023 07:41 by lmm")
+                                                             (* ; "Edited 14-Jan-87 12:46 by smL")
+
+(* ;;; "")
+
+(* ;;; "Return the name of the curren workstation.  Avoid consing up a new string if possible.")
+
+(* ;;; "")
+
+    (* ;; "")
+
+    (* ;; "The cached value in *WHO-LINE-HOST-NAME* gets invalidated by an entry on the list of \SYSTEMCACHEVARS")
+
+    (* ;; "")
+
+    (DECLARE (GLOBALVARS *WHO-LINE-HOST-NAME*))
+    (IF *WHO-LINE-HOST-NAME*
+        THEN *WHO-LINE-HOST-NAME*
+      ELSE (SETQ *WHO-LINE-HOST-NAME* (UNIX-GETENV "NAME"])
```

Comment:
"NAME" is not a standard environment variable denoting the hostname of the machine you're running on.  `(UNIX-GETPARM "HOSTNAME")` is the approved method for determining, for a Unix environment, what the host name is.